package.json

```diff
@@ -95,13 +95,13 @@
     "graphql-codegen-fragment-matcher": "^0.18.2",
     "husky": "^4.2.5",
     "jasmine": "^3.5.0",
-    "jasmine-core": "3.5.0",
+    "jasmine-core": "^3.7.1",
     "jasmine-spec-reporter": "4.2.1",
-    "karma": "^4.4.1",
-    "karma-chrome-launcher": "3.1.0",
+    "karma": "^6.3.2",
```

Comment:
Why are these versions modified?

Reply:
Oops I think I accidentally pushed code on the wrong  branch while trying to reproduce the build error.
Let me revert that change.

Reply:
Thanks! I don't think the revert undid this version change, though